<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parfüm Koleksiyonu Yönetimi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #1e293b;
}
QLabel {
    font-size: 14px;
    font-weight: bold;
    color: #f8fafc;
}
QLineEdit, QTextEdit, QComboBox, QDoubleSpinBox, QDateEdit {
    border: 1px solid #475569;
    border-radius: 6px;
    padding: 8px;
    background-color: #334155;
    color: #f8fafc;
    font-size: 14px;
    min-height: 30px;
}
QLineEdit:focus, QTextEdit:focus, QComboBox:focus {
    border: 2px solid #3b82f6;
}
QPushButton {
    background-color: #3b82f6;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 10px 18px;
    font-weight: bold;
    min-width: 120px;
    min-height: 36px;
    font-size: 14px;
}
QPushButton:hover {
    background-color: #2563eb;
}
QPushButton:pressed {
    background-color: #1d4ed8;
}
QTableView {
    border: 1px solid #475569;
    border-radius: 6px;
    background-color: #1e293b;
    selection-background-color: #3b82f6;
    selection-color: white;
    alternate-background-color: #273549;
    gridline-color: #475569;
    color: #f8fafc;
    font-size: 14px;
}
QTableView::item {
    padding: 8px;
    border-bottom: 1px solid #334155;
}
QHeaderView::section {
    background-color: #0f172a;
    color: #f8fafc;
    padding: 10px;
    border: none;
    border-right: 1px solid #334155;
    font-weight: bold;
    font-size: 14px;
}
QSplitter::handle {
    background-color: #475569;
    width: 2px;
}
QScrollBar:vertical {
    border: none;
    background: #334155;
    width: 12px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #64748b;
    min-height: 30px;
    border-radius: 6px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: none;
    background: none;
    height: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">font-size: 26px;
font-weight: bold;
color: #3b82f6;
margin: 10px;
padding: 20px;</string>
      </property>
      <property name="text">
       <string>PARFÜM KOLEKSİYONU YÖNETİMİ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="searchLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <widget class="QLabel" name="searchLabel">
        <property name="styleSheet">
         <string notr="true">font-size: 16px;</string>
        </property>
        <property name="text">
         <string>Parfüm Ara:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="aramaLineEdit">
        <property name="styleSheet">
         <string notr="true">font-size: 16px; padding: 10px;</string>
        </property>
        <property name="placeholderText">
         <string>Parfüm adı, markası veya koku notaları...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="yenileButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 16px;</string>
        </property>
        <property name="text">
         <string>Yenile</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="parfumTableView">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14px;</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderResizeMode">
        <enum>QHeaderView::Interactive</enum>
       </attribute>
      </widget>
      <widget class="QScrollArea" name="formScrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="spacing">
          <number>12</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="adLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Parfüm Adı:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="adLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="markaLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Marka:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="markaComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="turLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Tür:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="turComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="boyutLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Boyut (ml):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="boyutSpinBox">
           <property name="suffix">
            <string> ml</string>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="kalanMiktarLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Kalan Miktar (ml):</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="kalanMiktarSpinBox">
           <property name="suffix">
            <string> ml</string>
           </property>
           <property name="maximum">
            <double>1000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="tarihLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Edinme Tarihi:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDateEdit" name="tarihDateEdit">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="fiyatLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Fiyat:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="fiyatSpinBox">
           <property name="suffix">
            <string> ₺</string>
           </property>
           <property name="maximum">
            <double>100000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="kokuNotlariLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Koku Notaları:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QTextEdit" name="kokuNotlariTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="sezonLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Sezon Önerisi:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="sezonComboBox"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="durumLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Durum Önerisi:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QComboBox" name="durumComboBox"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="aciklamaLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Açıklama:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QTextEdit" name="aciklamaTextEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="fotoLabel">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Şişe Fotoğrafı:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <layout class="QHBoxLayout" name="fotoLayout">
           <item>
            <widget class="QLineEdit" name="fotoPathLineEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fotoBrowseButton">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">min-width: 80px;</string>
             </property>
             <property name="text">
              <string>Gözat</string>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="12" column="0" colspan="2">
          <widget class="QLabel" name="foto_preview">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 1px dashed #ccc;
background-color: #f9f9f9;</string>
           </property>
           <property name="text">
            <string>Fotoğraf yok</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <property name="spacing">
       <number>15</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ekleButton">
        <property name="styleSheet">
         <string notr="true">background-color: #4CAF50; min-width: 140px;</string>
        </property>
        <property name="text">
         <string>EKLE</string>
        </property>
        <property name="icon">
         <iconset theme="list-add"/>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guncelleButton">
        <property name="styleSheet">
         <string notr="true">background-color: #FFA000; min-width: 140px;</string>
        </property>
        <property name="text">
         <string>GÜNCELLE</string>
        </property>
        <property name="icon">
         <iconset theme="document-save"/>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="silButton">
        <property name="styleSheet">
         <string notr="true">background-color: #F44336; min-width: 140px;</string>
        </property>
        <property name="text">
         <string>SİL</string>
        </property>
        <property name="icon">
         <iconset theme="edit-delete"/>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="temizleButton">
        <property name="styleSheet">
         <string notr="true">background-color: #9E9E9E; min-width: 140px;</string>
        </property>
        <property name="text">
         <string>TEMİZLE</string>
        </property>
        <property name="icon">
         <iconset theme="edit-clear"/>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="actionCikis"/>
   </widget>
   <widget class="QMenu" name="menuYardim">
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionHakkinda"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuYardim"/>
  </widget>
  <action name="actionCikis">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="actionHakkinda">
   <property name="text">
    <string>Hakkında</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui> 